packages/rocketchat-statistics/server/functions/getUsages.js

```diff
@@ -0,0 +1,170 @@
+
+
+export function getUsages() {
+	const db = RocketChat.models.Users;
+	const usages = db.model.aggregate([
+		{
+			$lookup:
```

Comment:
If you want to start from the `user` collection and join the `messages` , you'll have to lookup to a temporary collection of messages, which is filtered by the `lastSend`. Filtering afterwards will cause horrible performance